Realm/RealmSwift/Realm.swift

```diff
@@ -83,6 +83,16 @@ public class Realm {
         }
     }
 
+    // MARK: Writing a Copy
+
+    public func writeCopyToPath(path: String, error: NSErrorPointer) {
+        rlmRealm.writeCopyToPath(path, error: error)
+    }
+
+    public func writeCopyToPath(path: String, encryptionKey: NSData, error: NSErrorPointer) {
```

Comment:
That would mean you'd have to explicitly pass in a `nil` encryption key when copying unencrypted realms.

We could have two methods:

```swift
writeCopyToPath(path: String, error: NSErrorPointer = nil)
writeCopyToPath(path: String, encryptionKey: NSData, error: NSErrorPointer = nil)
```